setup.cfg

```diff
@@ -26,8 +26,8 @@ setup_requires =
     pytest-runner
     setuptools-scm
 install_requires =
-    boto3 == 1.12.7
-    SQLAlchemy == 1.3.13
+    boto3 >=1.12.7,<1.14
+    SQLAlchemy >=1.3.13,<1.4
```

Comment:
It's again

Reply:
I figure, micro versions are meant to fix bugs, but minor versions can introduce features, which aren't supposed to break anything but might. So I set a "max version" of 1.3.9999999999 or so, by saying it has to be less than version 1.4.

Reply:
OK, I just understood your thought.